services/api-server/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.4.4
+current_version = 0.5.0
```

Comment:
If possible, it would be nice to get this from the `VERSION` file (or vice versa)